modules/ocf_printhost/files/enforcer-pc

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+gs -o /dev/null -sDEVICE=bbox $1 2>&1 | grep HiResBoundingBox | wc -l
```

Comment:
* Can you make the shebang `#!/bin/bash -eu` and add `set -o pipefail` as the second line of the file?
* Double quotes around `$1` just in case it has spaces